src/index.js

```diff
@@ -1,8 +1,10 @@
+import 'bootstrap/dist/css/bootstrap.css';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import './styles/custom.scss';
```

Comment:
change to `import './style/custom.scss'`

Reply:
fixed